lib/checks/forms/fieldset.js

```diff
@@ -50,28 +50,62 @@ function checkARIAGroup(group, name) {
 	return true;
 }
 
+function checkTable (group, nodes) {
```

Comment:
Shouldn't we also check that the table is a data table and has headers for the cells that contain the inputs?

Do we require a table marked up in this fashion to have a `<caption>`?